pyanaconda/ui/cockpit/anaconda-webui/test/ks.cfg

```diff
@@ -0,0 +1,3 @@
+%pre --log=/tmp/kickstart_pre.log
+/usr/libexec/cockpit-ws --no-tls --port 9090 --local-session=cockpit-bridge > /dev/null 2>&1 < /dev/null &
```

Comment:
Is it important to completely ignore the output? Won't that make debugging errors quite hard?

Reply:
Should I sent it to some file instead?

Reply:
Depends -- if the output goes to the journal or other visible place (e.g. Anaconda even has a VT for the journal, doesn't it?), it seems fine to just log to stderr. If it just disappears, or interferes with anaconda itself, then directing it to e.g. /run/cockpit.log (or /var/log/ if that doesn't end up in the installed system) is also fine.